osbs/api.py

```diff
@@ -80,6 +80,20 @@ def catch_exceptions(*args, **kwargs):
 LogEntry = namedtuple('LogEntry', ['platform', 'line'])
 
 
+def validate_arrangement_version(arrangement_version):
+    """Validate if the arrangement_version is supported
+
+    Shows a warning when version is deprecated
+
+    :param int arrangement_version: version to be validated
+    :raises ValueError: when version is not supported
+    """
+    if arrangement_version <= 5:
+        # TODO: raise as ValueError in release 0.54+
+        logger.warning("arrangement_version <= 5 is deprecated and will be removed"
```

Comment:
Do we want issue `warnings.warn` instead of `logger.warning` given that this is executed in koji/builders?